hack/kcp-edge.sh

```diff
@@ -0,0 +1,253 @@
+
+        elif [ $user_type == "kit" ]; then
+            # Apply CRDs
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/edge.kcp.io_customizers.yaml  &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/edge.kcp.io_edgeplacements.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/edge.kcp.io_edgesyncconfigs.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/edge.kcp.io_singleplacementslices.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/edge.kcp.io_syncerconfigs.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/meta.kcp.io_apiresources.yaml &> /dev/null
+
+            # Apply Exports
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiexport-edge.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiexport-meta.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-apiresources.meta.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-customizers.edge.kcp.io.yaml  &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-edgeplacements.edge.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-edgesyncconfigs.edge.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-singleplacementslices.edge.kcp.io.yaml &> /dev/null
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-syncerconfigs.edge.kcp.io.yaml &> /dev/null
+            echo "Finished populate the espw with kcp edge crds and apiexports"
+        else
+            echo "Unknown user type ..."
+            exit 1
+        fi 
+   fi
+fi
+
+sleep 5
+
+mailbox-controller --inventory-context=root --mbws-context=base -v=2 >& mailbox-controller-log.txt &
+
+run_status=$(wait_for_process mailbox-controller)
+if [ $run_status -eq 0 ]; then
+    echo " mailbox-controller is running (log file: mailbox-controller-log.txt)"
+else
+    echo " mailbox-controller failed to start ..... exiting"
+    sleep 2
+    exit
+fi
+
+
+# Start the edge-mc scheduler
+sleep 3
+scheduler -v 2 --root-user kcp-admin  --root-cluster root  --sysadm-context system:admin  --sysadm-user shard-admin >& edge-scheduler-log.txt &
```

Comment:
The scheduler normally works without any of those long flags, right?

Reply:
yes, fixed!